src/Redux/books/API.js

```diff
@@ -0,0 +1,20 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const POST_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/abc123/books';
```

Comment:
- Kindly, your post URL is not correct because unique identifier so please create your own unique identifier to make functional the get, post and delete methods 👍 
-  to create that unique identifier send a post request by Postman or other tools with `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/` URL then you will get your own unique identifier so replace the `abc123` with your unique identifier read the guideline for [more info](https://microverse.notion.site/Bookstore-API-51ea269061f849118c65c0a53e88a739)👍